packages/terra-list/src/terra-dev-site/test/list/ListItem.test.jsx

```diff
@@ -3,33 +3,37 @@ import React from 'react';
 import List, { Item } from 'terra-list/lib/index';
 
 const ListItemTest = () => (
-  <List role="listbox">
-    <Item
-      key="default"
-    >
-      <p>test default</p>
-    </Item>
-    <Item
-      key="chevron"
-      hasChevron
-    >
-      <p>test chevron</p>
-    </Item>
-    <Item
-      key="selectable"
-      isSelectable
-      id="selectable-item"
-    >
-      <p>test selectable</p>
-    </Item>
-    <Item
-      key="selected"
-      isSelectable
-      isSelected
-    >
-      <p>test selected</p>
-    </Item>
-  </List>
+  <React.Fragment>
+    <List>
+      <Item
+        key="default"
+      >
+        <p>test default</p>
+      </Item>
+      <Item
+        key="chevron"
+        hasChevron
+      >
+        <p>test chevron</p>
+      </Item>
+    </List>
+    <List role="listbox">
```

Comment:
why do we need to add a different role here? Is this needed for selectable lists?

Reply:
Correct, selectable lists need the listbox role added to the `<List />` component. [We note that in List readme.](https://github.com/cerner/terra-core/blob/b9d9846485813b17eb8fa581ea757c1445e2fba6/packages/terra-list/docs/List.md#terra-list)